ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -7,6 +7,7 @@
 
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
+import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
```

Comment:
not deleting this line? it won't crash but still incorrect to import it, may be accidentaly used again in the future